doc/changes/2466.bugfix

```diff
@@ -0,0 +1,4 @@
+Fixed issue #1919
+Dicke density state matrix should only have positive eigenvalues, but rounding errors causes eigenvalues of 0 to instead be represented as a very small negative number.
+This causes dicke_trace_function to return -Inf, since scipy.special.entr returns -Inf for any values < 0.
+This issue was fixed by simply taking the absolute value of the eigenvalues in dicke_trace_function.
```

Comment:
The towncrier entry is used to create the changelog. It does not need technical details or reference to issues numbers.

Reply:
Thank you for the comment! Hopefully the towncrier entry is good now